packages/algob/test/helpers/environment.ts

```diff
@@ -64,7 +64,7 @@ export function getEnv (defaultNetworkCfg?: NetworkConfig): AlgobRuntimeEnv {
     debug.enable("algob*");
   }
 
-  const config = loadConfigAndTasks(runtimeArgs);
+  const config = await loadConfigAndTasks(runtimeArgs);
```

Comment:
again, if we won't use `async / await` in config, then everything will be simpler.